spec/support/shared_examples/address_balances.rb

```diff
@@ -129,6 +129,30 @@
       end
     end
   end
+
+  describe '#transactions' do
```

Comment:
This shared example is call `address_balances`, can we create a `address_transactions` shared example?

Reply:
Ok

Reply:
sure, let me do so. thanks!